packages/next-app/src/app/internalmanager/(components)/TokenSelect.tsx

```diff
@@ -312,85 +310,44 @@ function TokenModal({
 function TokenRow({
   token,
   operationKind,
-  close,
   chainName,
 }: {
   token: TokenItem;
   operationKind: string;
-  close?: () => void;
   chainName: string;
 }) {
+  const router = useRouter();
   return (
-    <Table.BodyRow key={token.tokenAddress} classNames="hover:bg-blue4">
+    <Table.BodyRow
+      key={token.tokenAddress}
+      classNames="hover:bg-blue4 hover:cursor-pointer"
+      onClick={() => {
+        router.push(
+          `/internalmanager/${chainName}/${operationKind}/token/${token.tokenAddress}`
+        );
```

Comment:
NextJS' docs recommends using the Link component when there are no strict requirements for using `router.push`: https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating. Did you find this performed better than using Link? It should literally be the same thing

Reply:
😬 interesting! I think we could use TW's `table` classes to go around this, just FYI though. https://tailwindcss.com/docs/display#table